app/assets/stylesheets/components/_section.scss

```diff
@@ -8,3 +8,15 @@ section {
   padding: 1rem;
   width: 100%;
 }
+
+.news {
```

Comment:
I would create a new partial for news (_news.scss) and include in application.scss